backend/src/test/java/com/google/models/TimestampTest.java

```diff
@@ -70,4 +70,16 @@ public void equals_workingComparator() {
     assertFalse(timestamp.equals(null));
   }
 
+  @Test
+  public void compareTo_workingComparator() {
+    Timestamp baseTimestamp = new Timestamp(DAY_CONST, MONTH_CONST, YEAR_CONST);
+    Timestamp sameTimestamp = new Timestamp(DAY_CONST, MONTH_CONST, YEAR_CONST);
+    Timestamp largerTimestamp = 
+        new Timestamp(DAY_CONST + 1, MONTH_CONST, YEAR_CONST);
+
+    assertEquals(0, baseTimestamp.compareTo(sameTimestamp));
+    assertTrue(baseTimestamp.compareTo(largerTimestamp) <= 1);
```

Comment:
should this be <= -1? return a negative value if date is less than the other date

Reply:
ah yes, i was kind of tired last night LOL

my bad